mpas_analysis/ocean/time_series_ohc.py

```diff
@@ -106,11 +107,25 @@ def setup_and_check(self):  # {{{
         regions = [regions[index] for index in regionIndicesToPlot]
 
         for region in regions:
-            filePrefix = 'ohc_{}_{}.png'.format(region, mainRunName)
+            filePrefix = 'TAnomalyZ_{}_{}'.format(region, mainRunName)
             self.xmlFileNames.append('{}/{}.xml'.format(self.plotsDirectory,
-                                                        filePrefix))
-            self.filePrefixes[region] = filePrefix
+                                                            filePrefix))
+            self.filePrefixes[0, region] = filePrefix
```

Comment:
@gstreletz or @milenaveneziani, this is a clever extension of my `filePrefixes` dictionary.  However, it kind of defeats the purpose of using a dictionary if your keys are (or include) indices.  I would suggest adding a loop like:
```python
for region in regions:
    for filePrefix in ['TAnomalyZ', 'SAnomalyZ', 'OHCAnomalyZ', 'OHCAnomaly']:
        fullFilePrefix = '{}_{}_{}'.format(filePrefix, region, mainRunName)
        self.xmlFileNames.append('{}/{}.xml'.format(self.plotsDirectory, fullFilePrefix))
        self.filePrefixes[filePrefix, region] = fullFilePrefix
```
The naming is confusing so it might be worth rethinking the name of either the dictionary or the first key.

By the way, I had no idea you could use multiple keys on a dictionary.  That's very handy.  I've always used tuples, which I think is exactly what happens in this case, but it's a little more cluttered my way.